samples/extension-template/README.md

```diff
@@ -14,13 +14,13 @@ to define a sample [```EventHandler```](../../alfresco-java-event-api/alfresco-j
 
 To properly build and run the project in a local environment it is required to have installed some tools.
 
-* Java 17:
```

Comment:
Need to update the java version in the other read file as well.

Reply:
Updated the java version to 21 in readme file